packages/react/src/components/tag.tsx

```diff
@@ -1,11 +1,15 @@
+"use client"
+
 import { Slot } from '@radix-ui/react-slot'
 import clsx from 'clsx'
-import { AriaAttributes, HTMLAttributes } from 'react'
+import { XIcon } from 'lucide-react'
+import { AriaAttributes, HTMLAttributes, useState } from 'react'
 
 export type TagVariant = 'readonly' | 'dismissible' | 'operational' | 'selectable'
-export type SizeVariant = 'small' | 'medium' | 'large'
+export type SizeVariant = 'sm' | 'md' | 'lg'
+export type TagSeverity = 'error' | 'success' | 'warning' | 'info' | 'caution-minor' | 'caution-major' | 'caution-undefined'
```

Comment:
Tag severity is more application/business logic specific, I suggest removing severity from the tag component but then create a seperate component inside of the web application to apply this on top of the `Tag` component.

Reply:
If it simply to portray different colors I think we are fine just using the color names instead, e.g.

```
type TagVariant = 'blue' | 'red' ...
```